schema/workerschema/v1.json

```diff
@@ -254,6 +254,19 @@
           "response": {
             "$ref": "ClientWithSecret"
           }
+        },
+        "Revoke": {
+          "id": "dex.Client.Revoke",
+          "description": "Revoke all refresh tokens issues to the client for the authenticated user.",
+          "httpMethod": "DELETE",
+          "path": "client/refresh/{clientid}/revoke",
```

Comment:
why do you need the revoke at the end? Also, this seems to more rightly belong under account no?